WorkloadManagementSystem/Executor/Base/OptimizerExecutor.py

```diff
@@ -1,182 +1,183 @@
 """ Base class for all the executor modules for Jobs Optimization
 """
 
+
+__RCSID__ = "$Id$"
+
 import threading
-#Because eval(valenc) might require it
-import datetime
-from DIRAC import S_OK, S_ERROR, gLogger
+# Because eval(valenc) might require it
```

Comment:
this comment belongs to `import datetime`, probably means `import datetime` should be kept